packages/ui/src/pages/Home/MultisigActionMenu.tsx

```diff
@@ -45,30 +49,49 @@ const MultisigActionMenu = ({ withSendButton = true }: MultisigActionMenuProps)
         onClick: () => setIsChangeMultiModalOpen(true)
       })
 
+    // If we are NOT rendering "new transaction button" and is it's NOT a Watched account,
+    // the "Send transaction" item will appear in the list menu
+    if (!withNewTransactionButton && !selectedIsWatched) {
+      opts.unshift({
+        text: 'Send transaction',
+        icon: <HiOutlinePaperAirplane size={20} />,
+        onClick: () => setIsSendModalOpen(true)
+      })
+    }
+
```

Comment:
I don't think this is needed, since there's nothing in the 3 dots buttons that should change.
```suggestion
```

Reply:
Commented for the individual multisig button issue